### ISSUE TITLE
Not execute

### ISSUE THREAD

**Issue by Guslloween** [Innoxia/liliths-throne-public#1815]
**Describe the bug**
I was playing and suddenly the game froze and closed, when I tried to reopen it it wouldn't open, when I looked at the error document I discovered this:

**Expected behavior**
The game execute

**Attach files**

[error.log](https://github.com/Innoxia/liliths-throne-public/files/14448462/error.log)
[error.log](https://github.com/Innoxia/liliths-throne-public/files/14448531/error.log)

**Additional information:**
 - Game version 4.9
 - OS: Windows 10
 - Using .jar or .exe: both, .jar and .exe
- If using .jar, and you are not including an error.log file/text, please include your java version: 1.8.0_401 (Oracle Corporation)

**Comment by ac20011** [comment]
I checked the log and apparently it's because your system date is "February 29th". 
"2019" which i recall is the base year in the game isn't a leap year therefore "February 29, 2019" isn't possible. I set my system date to the 28th momentarily and that seemed to fix it.
I thought it was kind of fishy that someone had the exact same issue as me on the exact same day I decided to open the game again. Interesting that this bug might've flown under the radar because it can only happen once every 4 years

**Comment by Guslloween** [comment]
Yes! A pretty cool bug, which demonstrates the limitations of a system for dates. The first time I noticed it, four years ago, I ignored it, this time, I thought it had been fixed lol. Now it is working, normal calendar.

It was cute and I decided to share it.

**Comment by Vinom07** [comment]
See also https://github.com/Innoxia/liliths-throne-public/issues/1816.
